src/System.Management.Automation/CoreCLR/CorePsStub.cs

```diff
@@ -1207,20 +1207,20 @@ public RegistryValueKind GetValueKind(string name)
 
         public void Close()
         {
-            throw new NotImplementedException("Close() is not implemented. TransactedRegistry related APIs should not be hitten.");
+            throw new NotImplementedException("Close() is not implemented. TransactedRegistry related APIs should not be hidden.");
```

Comment:
... or `called`.

@vors if you suggest a replacement (`called`, `executed`, `hit`, `run`, or `used`), I can fix it up (or I can split it off into its own PR if that helps this one merge faster).